database.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+
+URL_DATABASE = 'postgresql://postgres:raees@localhost:5432/StockQuote'
```

Comment:
user name i.e. `postgres` and password  i.e.  `raees` for the Database should come from the environment variable.

You can create .env file to store sensitive credentioals, API keys, etc and load the values from the env in the application wherever needed.

Reply:
Done. Should I add the .env file to this repo too?